addons/website_sale/views/product_views.xml

```diff
@@ -200,7 +200,7 @@
         <field name="arch" type="xml">
             <field name='currency_id' position='after'>
                 <field name="selectable"/>
-                <field name="website_id" options="{'no_create': True}" string="Allow to use on " placeholder="None website"/>
+                <field name="website_id" options="{'no_create': True}" string="Allow to use on "/>
```

Comment:
do we still need label ?

Reply:
Alright, done, it will use the fields `string` as label, I added the uppercase on Website.